rdr_service/genomic/genomic_job_components.py

```diff
@@ -1694,6 +1697,63 @@ def calculate_contamination_category(self, sample_id, raw_contamination, member:
 
         return contamination_category
 
+    def _set_metrics_data_file_paths(self, row: dict):
+        cvl_site_bucket_map = config.getSettingJson(config.GENOMIC_CVL_SITE_BUCKET_MAP)
```

Comment:
At this point in the pipeline, you'll want to use the GCs' buckets, not the CVLs'. CVLs are downstream of the AW2. The CVL site map replaces BI with CO, i.e. Broad is not a CVL and Color is not a GC.

Reply:
This has been revised to use a new GENOMIC_GC_SITE_BUCKET_MAP configuration setting